src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -99,6 +105,28 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
             throw new Exception("Invalid return URL");
         }
 
+        [HttpGet]
+        public async Task<IActionResult> Logout(string logoutId)
+        {
+            return await Logout(new LogoutViewModel
+            {
+                LogoutId = logoutId
+            });
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
```

Comment:
Could this cause us problems when we integrate the Node.js front-end? (Although we obviously need to mitigate the CSRF threat.)

Reply:
At the moment no client should be calling this endpoint. So I will remove it.